tasks/configubuntu.yml

```diff
@@ -0,0 +1,39 @@
+# roles/mariadb/tasks/configubuntu.yml
```

Comment:
This file introduces a lot of code duplication. Is this necessary? What's the difference between this file and the original `config.yml`? Maybe adding `when: ansible_os_family == "RedHat"` or `when: ansible_os_family == "Debian"` to the necessary tasks is sufficient?